recipes/rest-pandas/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "rest-pandas" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "cdfbc1baa5927f833e986b586f751b677a38d0360d827291937ee68a0ed7e105" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+    - djangorestframework >=3.3.1
+    - pandas >=0.19.0
+    
+about:
+  home: https://github.com/wq/django-rest-pandas
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
Can you open an issue or a PR upstream for packaging the LICENSE file?

Reply:
done